app/scripts/dimApp.i18n.js

```diff
@@ -135,7 +135,7 @@
             SaveAsNew: "Save As New",
             Cancel: "Cancel",
             ItemsWithIcon: "Items with this icon will be equipped.",
-            ClickToEquip: "Click on an item toggle equip.",
+            ClickToEquip: "Click on an item toggle to equip.",
```

Comment:
Ah, I didn't recognize it broken up into two sentences. What's a better phrasing for "Click on an item to toggle whether or not it will be equipped."?